### PR TITLE
Add action type to link info

### PR DESCRIPTION
**Pull request by Luzagar**
minor pr to add link action type and rename actor loader to be more accurate

### CODE REVIEW

**Comment by fig02** [comment]
This is pretty cool.
In theory we have all the proc names already in `mProcInitTable` in `d_a_player_main`. In theory one could make a data table similar to what we did with the actor proc names

**Comment by fig02** [comment]
Understandable if you dont want to do that though :D just think it could be cool to print names instead of numbers eventually

**Comment by Luzagar** [comment]
not sure where to move move the link info panel somewhere else

File: modules/boot/src/actor_proc_data.cpp [hunk 1]
```diff
@@ -20,4 +20,9 @@ KEEP_FUNC void loadActorName() {
         int offset = (g_currentActor->mBase.mProcName * 32);
         loadFile("twwgz/procs.bin", &g_procData, sizeof(g_procData), offset);
     }
+}
+
+KEEP_FUNC void loadProcInit(int& i_procName) {
+    int offset = i_procName * 32;
+    loadFile("twwgz/procsInit.bin", &g_procInit, sizeof(g_procInit), offset);
```
Comment (fig02) [inline modules/boot/src/actor_proc_data.cpp]: I would name these after "player proc" or "player actions" so its more clear what these are, but not a big deal

**Review by fig02 — APPROVED** [review]

**Pull request merged**